src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3485,6 +3485,92 @@ private IOperation VisitNoneOperationExpression(IOperation operation)
             return Operation.CreateOperationNone(semanticModel: null, operation.Syntax, operation.ConstantValue, childrenBuilder.ToImmutableAndFree(), operation.IsImplicit);
         }
 
+        public override IOperation VisitInterpolatedString(IInterpolatedStringOperation operation, int? captureIdForResult)
+        {
+            var partsBuilder = ArrayBuilder<IInterpolatedStringContentOperation>.GetInstance(operation.Parts.Length);
+            ArrayBuilder<IOperation> formatBuilderOpt = null;
```

Comment:
>ArrayBuilder<IOperation> formatBuilderOpt = null; [](start = 12, length = 49)

Consider getting rid of this builder. It really doesn't matter when to visit a literal, we can visit it in the second loop.

Reply:
Sure, I will also move the visit for the interpolation text part to the second loop.

---
In reply to: [185963061](https://github.com/dotnet/roslyn/pull/26545#discussion_r185963061) [](ancestors = 185963061)